cmd/velero-restic-restore-helper/main.go

```diff
@@ -75,3 +76,26 @@ func done() bool {
 
 	return true
 }
+
+// cleanupDoneFiles returns if it fails to remove all of the `.velero` dirs
+// created by Velero when a restic restore completes or when all directories
+// are removed. It does not return an error to run a 'soft' cleanup.
+func cleanupDoneFiles() {
+	children, err := ioutil.ReadDir("/restores")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "ERROR reading /restores directory: %s\n", err)
+		return
+	}
+
+	for _, child := range children {
+		if !child.IsDir() {
+			fmt.Printf("%s is not a directory, skipping.\n", child.Name())
+			continue
+		}
+		doneFileDir := filepath.Join("/restores", child.Name(), ".velero")
+		if err := os.RemoveAll(filepath.Join(doneFileDir, ".velero")); err != nil {
```

Comment:
looks like there's an extra `.velero` in the path here (it's being added in both L95 and L96).